### PR TITLE
fix: modified get_public_name to show redactions on participant team views

### LINKED ISSUE #2249: Redactions not reflected in team views
Whenever a person tries to view an individual team's record, the full names of all speakers are shown, ignoring redactions.

### PR DESCRIPTION
**Pull request by Veyron2121**
@tienne-B anything I should specifically include in this description?

Fixes #2249

### CODE REVIEW

File: tabbycat/participants/models.py [hunk 1]
```diff
@@ -129,6 +130,9 @@ def __str__(self):
         return str(self.name)
 
     def get_public_name(self, tournament):
+        # This allows for jinja html templates to still access redactions from the tab
+        if self.anonymous:
+            return mark_safe("<em>Redacted</em>")
```
Comment (tienne-B) [inline tabbycat/participants/models.py:135]: "Redacted" should be marked for translation, like
```suggestion
            return mark_safe("<em>" + _("Redacted") + "</em>")
```
No need for that comment (it's also wrong as we don't use Jinja) :P

**Review by tienne-B — COMMENTED** [review]
Your PR description is fine; I often just say "look at the commit messages".

**Comment by philipbelesky** [comment]
Quick question, for my own curiosity, is the expectation now that redactions also apply to non-public pages? Originally it was a concession to the public / search-indexed nature of tab releases, but I guess it is now intended to cover all places where names are shown to participants?

**Comment by tienne-B** [comment]
Non-public like private URL-accessible pages? I'd think that keeping a simpler approach of admin vs non-admin would be better than also adding "restricted." We should check for the places that use `get_public_name()` as to not code/anonymize on admin pages.

**Comment by philipbelesky** [comment]
> Non-public like private URL-accessible pages?

Yea, that was what I was thinking. But to be clear, I don't have any preference about what should happen, I was just curious about what user expectations look like these days. The simpler implementation is definitely preferred!

**Comment by tienne-B** [comment]
Yeah, think the expectation is basically "remove all traces I was here"

**Pull request merged**